packages/shared/src/extension/chrome/background/account.ts

```diff
@@ -84,17 +85,18 @@ export async function handleSettingsLookup(
     DEFAULT_SETTINGS
   );
 
-  const experimentInfo = await readOrDefault('experiment.json', t.any, {});
+  if (settingsJson.publicKey?.length && settingsJson.secretKey?.length) {
+    log.info('Loaded configuration from file settings.json: %j', settingsJson);
+  }
 
+  const experimentInfo = await readOrDefault('experiment.json', t.any, {});
   const settings = { ...DEFAULT_SETTINGS, ...settingsJson, ...experimentInfo };
 
-  try {
-    if (!(settings.publicKey?.length && settings.secretKey?.length))
-      throw new Error('Invalid key material found in settings.json');
-
-    log.info('Loaded configuration from file settings.json: %j', settings);
-  } catch (err) {
-    log.info('Error caught while checking settings.json: %s', err);
+  if (!(settings.publicKey?.length && settings.secretKey?.length)) {
```

Comment:
I wouldn't add here this behavior cause the keys needs to be generated after we check also the settings stored in the `chrome.storage` and from here this function we should only return what's in those files (`settings.json` and `experiment.json`)

Reply:
thanks! I'll do a force push with a reset